src/vs/editor/common/viewLayout/linesLayout.ts

```diff
@@ -699,6 +705,7 @@ export class LinesLayout {
 				break;
 			}
 		}
+		linesOffsets[lineNumber - startLineNumber] = currentLineRelativeOffset;
```

Comment:
I also don't understand this line here. Was this a bug before?

Reply:
Sometimes I need the top offset of the next line requested. So I need the line offsets of all lines in the view, plus one the next line.